### ISSUE TITLE
Issue replicating the Mortgage demo

### ISSUE THREAD

**Issue by vivek-capiot** [entando/entando-core#534]
Just installed Entando 5.0
I am trying to replicate the mortgage application given in the documentation. Under section 3.1.2 I am not able to find the specified widgets in the app builder console. 
Is there something I am missing? Also how to get all the widgets that Entando provides internally?

**Comment by joewhite101** [comment]
Hi @vivek-capiot 

There is a bug in our docs as compared to the related available jars. This will be updated and resolved in our next patch which is 5.0.1. To add widgets that will work for the mortgage app for now add this to your pom:
 
```
<dependency>
            <groupId>org.entando.entando.bundles.pages</groupId>
            <artifactId>entando-page-inspinia</artifactId>
            <version>${entando.version}</version>
            <type>war</type>
        </dependency>
        <dependency>
            <groupId>org.entando.entando.bundles.pages</groupId>
            <artifactId>entando-page-light</artifactId>
            <version>${entando.version}</version>
            <type>war</type>
        </dependency>
```

And then use the "light" widgets instead of the purple ones. They are the same with the only difference being the color. One 5.0.1 is out I will provide an updated on this ticket and you can try the purple ones.

To get all of the widgets currently you will need to manually add them to your pom.xml file. The widgets available can be found under the entando-components project ( https://github.com/entando/entando-components) under plugins and ui-bundles.

We are working on a master pom that will contain all of the widgets and plugins we provide. This will be available in September.

**Comment by joewhite101** [comment]
Hi @vivek-capiot I believe this is resolved now. Let us know if you run into other problems.
